.github/workflows/create-release.yaml

```diff
@@ -62,10 +62,20 @@ jobs:
         GO_BUILD_TAGS=experimental
       tags: "${{ github.event.inputs.name }}-experimental"
 
+  unit-tests:
+    uses: ./.github/workflows/call-unit-lint.yaml
+
+  integration-tests:
```

Comment:
I'm just wondering if we also want to additionally test the experimental image? 🤔

Reply:
In daily it was agreed on that we keep it as it is for now.